src/main/java/org/jenkinsci/test/acceptance/plugins/credentials/FileCredentials.java

```diff
@@ -9,7 +9,7 @@
 @Describable("Secret file")
 public class FileCredentials extends BaseStandardCredentials {
 
-    public Control file = control("file");
+    public Control file = control(by.name("file"));
```

Comment:
Well, I really doubt we will have conflicts here since it is a very simple page the one that's rendered when creating a credential.

That said, I wasn't aware of that use of the `control` method, and it does seem like the most reasonable way to implement this. Very useful, thanks!